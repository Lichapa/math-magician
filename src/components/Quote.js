import React from 'react';

function Quote() {
  return (
    <div className="container">
      <h2 className="center">Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -Willam Paul Thurston</h2>
    </div>
  );
}
export default Quote;
