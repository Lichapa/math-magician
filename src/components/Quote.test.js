import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Home';

it('renders correctly', () => {
  const tree = renderer
    .create(<Quote page="http://localhost:3000/quote">Home</Quote>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
