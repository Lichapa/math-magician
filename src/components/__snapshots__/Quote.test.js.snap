// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio eum labore sequi. Qui dolorum enim deserunt minima ratione, consequuntur quas esse culpa aliquam rem, accusantium consectetur labore odio repellat consequatur?
  </p>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio eum labore sequi. Qui dolorum enim deserunt minima ratione, consequuntur quas esse culpa aliquam rem, accusantium consectetur labore odio repellat consequatur?
  </p>
</div>
`;
